backend/canisters/translations/impl/src/model/translations.rs

```diff
@@ -58,37 +62,43 @@ impl Translations {
     }
 
     pub fn approve(&mut self, id: u64, user_id: UserId, now: TimestampMillis) -> ApproveResponse {
-        if let Some(translation) = self.translations.get_mut(id as usize) {
+        let translation = if let Some(translation) = self.translations.get(id as usize) {
```

Comment:
You could use the relatively new `let-else` pattern here.

`let Some(translation) = self.translations.get(id as usize) else { return ApproveResponse::NotFound };`